test/test_images/performance/main.go

```diff
@@ -63,6 +76,8 @@ func init() {
 	// role=aggregator
 	flag.StringVar(&listenAddr, "listen-address", ":10000", "Network address the aggregator listens on.")
 	flag.UintVar(&expectRecords, "expect-records", 1, "Number of expected events records before aggregating data.")
+	flag.Var(&additionalTags, "additional-tags", "Array of environment specific additional tags. Example " +
```

Comment:
If you're going to use the multi-parameter style, I think it might make sense for the flag name to be singular, e.g. `--tag=tag1 --tag=tag2`